ui-core/src/participant/landing/sections/HeroWithImageTemplate.tsx

```diff
@@ -121,14 +168,14 @@ function HeroWithImageTemplate(props: HeroWithImageTemplateProps) {
               'd-flex justify-content-center align-items-center p-0'
             )}
           >
-            <ConfiguredMedia media={image} className="img-fluid"/>
+            {!imageAsBackground && <ConfiguredMedia media={image} className="img-fluid"/>}
           </div>
         )}
         <div
           className={classNames(
             'col-12', `col-lg-${12 - imageCols}`,
             'py-3 p-sm-3 p-lg-5',
-            'd-flex flex-column flex-grow-1 justify-content-around'
+            'd-flex flex-column flex-grow-1 justify-content-center'
```

Comment:
same as [matt's pr](https://github.com/broadinstitute/juniper/pull/1027); no other changes should have any visible impact to existing site contents